### PR TITLE
adding atomicInteger for cloud request

### PR DESCRIPTION
**Pull request by gvijayakuma2**
@Behnke19 @mark-a-johnson for review

### CODE REVIEW

**Comment by servicemesh-autobuild** [comment]
:white_check_mark: Build was successful :tada:


http://ci.servicemesh.com:8080/job/csc-agility-platform-sdk-prb/204/

File: agility.cloud.sdk/src/main/resources/schema/general.xsd [hunk 1]
```diff
@@ -94,6 +94,16 @@ ALL SOFTWARE, INFORMATION AND ANY OTHER RELATED COMMUNICATIONS (COLLECTIVELY, "W
 						</xsd:appinfo>
 					</xsd:annotation>
 				</xsd:element>
+				
+				<xsd:element name="atomicRequestID" type="agility:AtomicInteger">
```
Comment (Behnke19) [inline agility.cloud.sdk/src/main/resources/schema/general.xsd]: I'm not sure the type should be AtomicInteger. I think maybe the type should be long and we would use an AtomicInteger asset behind the scenes to generate the values.

Comment (gvijayakuma2) [reply agility.cloud.sdk/src/main/resources/schema/general.xsd]: Agree with you, in bridge we will use AtomicInteger asset for generating number and set to long type.

Comment (mark-a-johnson) [reply agility.cloud.sdk/src/main/resources/schema/general.xsd]: The type should be xsd:integer and the field should be named something without the "atomic" in the name. Like "cloudRequestId"

**Comment by servicemesh-autobuild** [comment]
:white_check_mark: Build was successful :tada:


http://ci.servicemesh.com:8080/job/csc-agility-platform-sdk-prb/205/

**Comment by servicemesh-autobuild** [comment]
:white_check_mark: Build was successful :tada:


http://ci.servicemesh.com:8080/job/csc-agility-platform-sdk-prb/206/

**Pull request merged**